src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int ONE_ELEMENT = 1;
```

Comment:
Agree. Confused name. 
```suggestion
    private static final int ONE = 1;
```

Reply:
done